base/coreir.jl

```diff
@@ -14,61 +14,75 @@ Core.Const
 """
     struct PartialStruct
         typ
-        undef::BitVector # represents whether a given field may be undefined
+        undefs::Vector{Union{Nothing,Bool}} # represents whether a given field may be undefined
```

Comment:
Renamed `undef` to `undefs` to avoid naming collision with the `undef` initializer.